app/views/shared/_navigation.html.erb

```diff
@@ -37,21 +37,20 @@
           </ul>
         </li>
 
-        <li>
-          <%= link_to "#", class: "menu_link header_button" do %>
-            <i class="fa fa-sliders"></i>
-            <p>Account Options</p>
-          <% end %>
-          <ul class="eui-sub-menu">
-            <li><%= link_to "Edit Account", edit_user_registration_path %></li>
-            <% if current_user.admin %>
-              <li><%= link_to "Create User", new_user_registration_path %></li>
-              <li id="record_refresh"><%= link_to "Refresh Records", record_refresh_path %></li>
+        <% if current_user.admin %>
```

Comment:
Maybe add ?

Reply:
So I'm going to leave this the way it is, as the comment above admin function says, think maybe create a separate ticket as technical debt to refactor this.
  # ==== Method
  # This method checks if the user is a "admin", which is a legacy role name. It's only
  # intended for backwards compatibility. For new code, use the 'role' attribute directly.
  def admin
    role.eql?("admin")
  end

Reply:
I added this to technical debt..   
https://bugs.earthdata.nasa.gov/browse/CMRARC-412